### PR TITLE
Add FOSSA scans

### PR DESCRIPTION
**Pull request by goatherder**
So we can catch licensing and security violations early

### CODE REVIEW

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -5,10 +5,15 @@ on:
     branches: '*'
 
 jobs:
+  fossa-scans:
+    name: Rune FOSSA Scans
```
Comment (MathiasTim) [inline .github/workflows/pull-request.yml:9]: Is the extra `e` a typo?

**Review by MathiasTim — APPROVED** [review]

File: .github/workflows/fossa-scan.yml [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+on:
+  workflow_call:
+
+jobs:
+    fossa-scan:
+        runs-on: ubuntu-latest
+        steps:
+          - name: "Checkout Code"
+            uses: actions/checkout@v3
```
Comment (enjikaka) [inline .github/workflows/fossa-scan.yml:11]: v4?

File: .github/workflows/fossa-scan.yml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+
+          - name: "Run FOSSA Scan"
+            uses: fossas/fossa-action@main
```
Comment (enjikaka) [inline .github/workflows/fossa-scan.yml:14]: is @main safe or do we wanna lock the version of it?

File: .github/workflows/pull-request.yml (hunk 1 above)
Comment (michpohl) [reply .github/workflows/pull-request.yml]: image

Comment (osmestad) [reply .github/workflows/pull-request.yml]: Rune was here! :-P

File: .github/workflows/fossa-scan.yml (hunk 3 above)
Comment (michpohl) [reply .github/workflows/fossa-scan.yml]: let's lock if we can, version or commit id

**Review by osmestad — APPROVED** [review]

**Review by enjikaka — APPROVED** [review]

**Pull request merged**